app/controllers/rapidfire/application_controller.rb

```diff
@@ -17,9 +17,5 @@ def authenticate_administrator!
     def scoped
       :user
     end
-
-    def current_user
```

Comment:
@emilsoman its generally defined by application. This method was resulting in infinite recursion!